src/common/users.ts

```diff
@@ -571,6 +573,18 @@ export const shouldAllowRestore = async (
   });
 };
 
+export const checkIfUserIsTeamMember = async (
+  con: DataSource,
+  userId: string,
+): Promise<boolean> => {
+  const isTeamMember = await con.getRepository(Feature).findOneBy({
```

Comment:
Maybe add specific select on value.

Reply:
Changed to only select value

Reply:
Though function isn't used that much for telemetry, since I moved that to the JWT instead. But still keeping it in since it's being re-used other places